modules/post/hardware/automotive/can_flood.rb

```diff
@@ -0,0 +1,43 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Post
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'can_flood',
```

Comment:
The module name allows whitespace and capitalization, and is usually slightly more descriptive.

`CAN Flood` or `CAN Bus Flood` or `CAN Interface Flood` perhaps ?

Reply:
Yes! I think "CAN Flood" is better! Thanks!